booster/library/Booster/JsonRpc.hs

```diff
@@ -113,13 +110,12 @@ respond stateVar =
                                     ]
                 Right (pat, substitution, unsupported) -> do
                     unless (null unsupported) $ do
+                        withKorePatternContext (KoreJson.KJAnd (externaliseSort $ sortOfPattern pat) unsupported) $
```

Comment:
I think I have an idea how to avoid duplicating the log message here. As I understand, we only have to have duplicates for warnings. Then we could create a special variant of the `logMessage` function, say `logWarning`, which would append `[warning]` at the tail of the context. We than make a static glob pattern `*warning` that is enabled by default.